quickstarts/python/sample.py

```diff
@@ -44,33 +64,45 @@ def get_user_timeofday():
         print("Entered value is invalid. Setting feature value to", time_features[0] + ".")
         res['time_of_day'] = time_features[0]
     return res
+# </createUserFeatureTimeOfDay>
 
+# <mainLoop>
 keep_going = True
 while keep_going:
-    eventid = uuid.uuid4()
+
+    eventid = str(uuid.uuid4())
+
     context = [get_user_preference(), get_user_timeofday()]
     actions = get_actions()
-    rank_request = RankRequest(actions,context_features=context,excluded_actions=['juice'],event_id=eventid)
-    response = client.rank(rank_request)
 
+    # <rank>
+    rank_request = RankRequest( actions=actions, context_features=context, excluded_actions=['juice'], event_id=eventid)
+    response = client.rank(rank_request=rank_request)
+    # <rank>
+    
     print("Personalizer service ranked the actions with the probabilities listed below:")
+    
     rankedList = response.ranking
     for ranked in rankedList:
         print(ranked.id, ':',ranked.probability)
 
     print("Personalizer thinks you would like to have", response.reward_action_id+".")
     answer = input("Is this correct?(y/n)\n")[0]
-    reward_val = 0.0
+
+    # <reward>
+    reward_val = "0.0"
```

Comment:
The value type is a float, so we don't want to instruct the user to use a string. Did you have runtime issues when not a string?

Reply:
Funny you should ask...yes, a float throws an error that a float is not one of the allowed types of number or string.